src/Microsoft.DotNet.Build.Tasks/PackageFiles/RepoAPI.Mapping.props

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Provides a default API implementation where possible by setting BuildTools properties/items.
+
+    API documentation: https://github.com/dotnet/source-build/tree/dev/release/2.0/Documentation/auto-dependency-flow
+  -->
+
+  <!-- Produce packages into the specified blob feed. -->
+  <PropertyGroup Condition="'$(DotNetOutputBlobFeedDir)' != ''">
+    <PackageOutputPath>$(DotNetOutputBlobFeedDir)packages/</PackageOutputPath>
+    <SymbolPackageOutputPath>$(DotNetOutputBlobFeedDir)assets/</SymbolPackageOutputPath>
+  </PropertyGroup>
+
+  <!--
+    Import the restore source props file if passed to get "DotNetRestoreSources".
+
+    Note the "!= ''" check rather than Exists. If this API argument is passed but the file doesn't
+    exist, something is wrong and an error should be bubbled up as soon as we know about it.
+  -->
+  <Import Project="$(DotNetRestoreSourcePropsPath)"
+          Condition="'$(DotNetRestoreSourcePropsPath)' != ''"/>
+
+  <!-- Use passed NuGet restore sources, if any. -->
+  <PropertyGroup>
+    <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
```

Comment:
Are projects who import this .props file expected to use `RestoreSources` at all?  If so, I think there is an issue here.

If the project sets `RestoreSources` before importing this file, this will overwrite that value.  If the project sets `RestoreSources` after importing this file, their values won't get added to `DotnetSourceList` below.

I would think this would append to the `RestoreSources` property instead of overwriting it.

Reply:
I'll change it to append. You're right, no reason to restrict people from defining sources before importing this file.

> If the project sets RestoreSources after importing this file, their values won't get added to DotnetSourceList below.

Experimentally, this isn't the case: items are all evaluated after properties. For clarity, it might be nice to move it into a target to make the ordering explicit, but some Core repos and parts of BuildTools depend directly on the static values.

Reply:
> items are all evaluated after properties

I keep forgetting about this...